asf_welsh_energy_consultation/analysis/produce_plots_and_stats.py

```diff
@@ -33,11 +33,31 @@
 
 
 if __name__ == "__main__":
+    # ======================================================
+    # Total MCS installations
+    enhanced_mcs = process_data.get_enhanced_combined(mcs_or_gold="mcs")
+    total_cumulative_installations = process_data.get_total_cumsums(
+        data=enhanced_mcs, installation_date_col="commission_date"
+    )
+
+    total_cumulative_installations_chart = time_series_comparison(
+        data=total_cumulative_installations,
+        title="Cumulative MCS certified heat pump installations over time",
+        y_var="cumsum:Q",
+        y_title="Number of heat pump installations",
+        color_var="colour:N",
+        filename="total_cumulative_installations",
+        output_dir=output_folder,
+    )
```

Comment:
No need to review this part - This is merged from branch 10 which has already been reviewed